modules/channel-web/src/backend/api.ts

```diff
@@ -297,8 +297,10 @@ export default async (bp: typeof sdk, db: Database) => {
       }
 
       await sendNewMessage(botId, userId, conversationId, payload, req.credentials)
-      await bp.dialog.deleteSession(userId)
-      res.status(200).send({})
+
+      const sessionId = await bp.dialog.createId({ botId, target: userId, threadId: conversationId, channel: 'web' })
+      await bp.dialog.deleteSession(sessionId)
+      res.send({})
```

Comment:
why not use res.sendStatus(200) instead of sending an empty body

Reply:
Good question, I thought there was a reason for it in the first place, so I just removed the superfluous 200 :-P

Reply:
yeah I'd send 200 instead of an empty object, because I'm pretty sure you'll get `{}` in your response.